order/order-service/src/test/java/shop/woowasap/order/service/support/fixture/OrderDtoFixture.java

```diff
@@ -0,0 +1,54 @@
+package shop.woowasap.order.service.support.fixture;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+import shop.woowasap.order.domain.Order;
+import shop.woowasap.order.domain.OrderProduct;
+import shop.woowasap.order.domain.in.response.OrderProductResponse;
+import shop.woowasap.order.domain.in.response.OrderResponse;
+import shop.woowasap.order.domain.in.response.OrdersResponse;
+
+public final class OrderDtoFixture {
+
+    private OrderDtoFixture() {
+        throw new UnsupportedOperationException("Cannot invoke constructor \"OrderDtoFixture()\"");
+    }
+
+    public static OrdersResponse from(final List<Order> orders, final String locale, final int page,
+        final int totalPage, final String fixedName) {
+
+        final List<OrderResponse> orderResponses = getOrderResponse(orders, locale, fixedName);
+
+        return new OrdersResponse(orderResponses, page, totalPage);
+    }
+
+    private static List<OrderResponse> getOrderResponse(final List<Order> orders, final String locale, final String fixedName) {
+        final List<OrderResponse> orderResponses = new ArrayList<>();
+
+        for (Order order : orders) {
+            final List<OrderProductResponse> orderProductResponses = getOrderProductResponse(order, fixedName);
+
+            final OrderResponse orderResponse = new OrderResponse(order.getId(),
+                orderProductResponses,
+                order.getTotalPrice().toString(), order.getOrderProducts().size(),
+                LocalDateTime.ofInstant(order.getCreatedAt(), ZoneId.of(locale)));
+
+            orderResponses.add(orderResponse);
+        }
+
+        return orderResponses;
+    }
+
+    private static List<OrderProductResponse> getOrderProductResponse(final Order order, final String fixedName) {
+        final List<OrderProductResponse> orderProductResponses = new ArrayList<>();
+        for (OrderProduct orderProduct : order.getOrderProducts()) {
+            final OrderProductResponse orderProductResponse = new OrderProductResponse(
+                orderProduct.getProductId(), fixedName);
+
+            orderProductResponses.add(orderProductResponse);
+        }
+        return orderProductResponses;
+    }
```

Comment:
```suggestion
    private static List<OrderResponse> getOrderResponse(final List<Order> orders, final String fixedName) {
        return orders.stream()
            .map(order -> {
                List<OrderProductResponse> orderProductResponses = getOrderProductResponse(order, fixedName);
                String totalPrice = order.getTotalPrice().toString();
                int productCount = order.getOrderProducts().size();
                LocalDateTime createdAt = LocalDateTime.ofInstant(order.getCreatedAt(), ZoneId.of(locale));
                return new OrderResponse(order.getId(), orderProductResponses, totalPrice, productCount, createdAt);
            })
            .toList();
    }

    private static List<OrderProductResponse> getOrderProductResponse(final Order order, final String fixedName) {
        return order.getOrderProducts().stream()
            .map(orderProduct -> new OrderProductResponse(orderProduct.getProductId(), fixedName))
            .toList();
    }
```
스트림을 이용해도 좋을 것 같습니다😃

Reply:
86cbb03

변경했습니다 👍